<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GCal4edX - MOOC TÃ©cnico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>107</height>
           </size>
          </property>
          <property name="title">
           <string>Generate Google Calendar</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Create Google Calendar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>121</height>
           </size>
          </property>
          <property name="title">
           <string>edX Course Export File</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>Studio -&gt; Tools -&gt; Export</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Course Display Name</string>
                 </property>
                 <property name="buddy">
                  <cstring>lineEdit_4</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>122</height>
           </size>
          </property>
          <property name="title">
           <string>Google Calendar Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Events Text Language</string>
                 </property>
                 <property name="buddy">
                  <cstring>comboBox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>en</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>pt</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Course Run (e.g. 2020_2)</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="toolTip">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Studio -&gt; Settings -&gt; Schedule &amp; Details -&gt; Course Run</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Events</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Events duration</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox">
                 <property name="maximum">
                  <number>12</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>H</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Deadline duration</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_2">
                 <property name="maximum">
                  <number>12</number>
                 </property>
                 <property name="value">
                  <number>6</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>H</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Override default events reminders</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Email Reminders Before</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_3">
                    <property name="maximum">
                     <number>168</number>
                    </property>
                    <property name="value">
                     <number>24</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>H</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Create all day events</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Google Account</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Log out and Close App</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>groupBox_6</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>createCalendarSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>advancedSettingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>advancedSettingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>advancedSettingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>advancedSettingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>advancedSettingsChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>logoutSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseSlot()</slot>
  <slot>settingsChangedSlot()</slot>
  <slot>createCalendarSlot()</slot>
  <slot>advancedSettingsChangedSlot()</slot>
  <slot>logoutSlot()</slot>
  <slot>openHyperlink(QString)</slot>
 </slots>
</ui>
